docs/index.rst

```diff
@@ -125,39 +100,141 @@ argument.
+any of the supported *fast* formats:
 
-This function will raise an exception if the input date is not in one of
-these allowed formats:
+- `secs`: CXC seconds
+- `date`: Year Day-of-Year
+- `greta`: GRETA format (input can be string, float or int)
+- `maude`: MAUDE format (input can be string or int)
+- `jd`: Julian Day (requires `fmt_in="jd"` to identity this format)
 
-- YYYY:DDD
-- YYYY:DDD:HH:MM
-- YYYY:DDD:HH:MM:SS
-- YYYY:DDD:HH:MM:SS.sss
+For example::
+
+  >>> from cxotime import convert_time_format
+  >>> convert_time_format("2022:001:00:00:00.123", "greta")
+  '2022001.000000123'
+  >>> convert_time_format(100.123, "date")
+ '1998:001:00:00:36.939'
+  >>> convert_time_format(2459580.5, "date", fmt_in="jd")
+  '2022:001:00:00:00.000'
+
+Note that this function can be used to convert between any of the supported |CxoTime|
+formats, but it will internally use a |CxoTime| object so the performance will not be
+improved. For example::
+
+    >>> convert_time_format(2022.123, fmt_out="date", fmt_in="frac_year")
+    '2022:045:21:28:48.000'
+
+    # Exactly equivalent to:
+    >>> CxoTime(2022.123, format="frac_year").date
+    '2022:045:21:28:48.000'
+
+Convenience functions like ``secs2date``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For historical compatibility and for succinct code, direct conversion between any two
+of the "fast" formats is also available via convenience functions. These have the name
+``<fmt_in>2<fmt_out>`` where ``fmt_in`` and ``fmt_out`` are the input and output formats.
+Examples include :func:`~cxotime.convert.date2secs`, :func:`~cxotime.convert.secs2greta`,
+and :func:`~cxotime.convert.greta2jd`.
+
+::
+    >>> from cxotime import secs2greta
+    >>> secs2greta([100, 1000])
+    array(['1998001.000036816', '1998001.001536816'], dtype='<U17')
 
-Conversely, for fast conversion from CXC seconds to Year Day-of-Year date, the
-function :func:`~cxotime.cxotime.secs2date` can be used. For scalar inputs this
-is 15-20 times faster than the equivalent call to ``CxoTime(secs).date``.
+
+Print common time conversions
+-----------------------------
+
+The `cxotime` package has functionality to convert a time betweeen a variety of common
+time formats. This convenience function is available in two ways, either as a
+command line script or as class method :meth:`~cxotime.CxoTime.print_conversions`::
+
+    $ cxotime 2022:002:12:00:00
+    local       2022 Sun Jan 02 07:00:00 AM EST
+    iso_local   2022-01-02T07:00:00-05:00
+    date        2022:002:12:00:00.000
+    cxcsec      757512069.184
+    decimalyear 2022.00411
+    iso         2022-01-02 12:00:00.000
+    unix        1641124800.000
+
+::
+
+    $ cxotime  # Print current time
+    local       2023 Tue Jan 10 01:41:02 PM EST
+    iso_local   2023-01-10T13:41:02.603000-05:00
+    date        2023:010:18:41:02.603
+    cxcsec      789763331.787
+    decimalyear 2023.02679
+    iso         2023-01-10 18:41:02.603
+    unix        1673376062.603
+
+::
+
+    $ python
```

Comment:
if you remove this, the code is rendered as python code, with proper colors, and the user can copy/paste.